multicluster/service-mirror/cluster_watcher.go

```diff
@@ -50,6 +50,8 @@ type (
 		eventsQueue             workqueue.RateLimitingInterface
 		requeueLimit            int
 		repairPeriod            time.Duration
+		alive                   bool
```

Comment:
might consider naming this to indicate it represents the liveness of the target gateway